external/metabolism/custcom.lua

```diff
@@ -13,30 +13,30 @@ dofile( dpath.."chatcore.lua" )
 dofile( dpath.."functions.lua" )
 local tokenize = Explode
 
-function chkpriv(user, n)
-	local profile = user.iProfile
+function chkpriv( tUser, n )
+	local profile = tUser.iProfile
 	if n == -1 then return true end
 	if profile == -1 then return false end
 	if profile > n then return false end
 	return true
 end
 
-function isHigherRanked(user, victim)
-	local userprofile = user.iProfile
-	local victimprofile = victim.iProfile
+function isHigherRanked( tUser, tVictim )
+	local userprofile = tUser.iProfile
+	local victimprofile = tVictim.iProfile
 	if victimprofile == -1 and userprofile ~= -1  then return true end
 	if userprofile < victimprofile then return true end
 	return false
 end
 
-function isthere(user, tabl)
-	return tabl[user]
+function isthere( sUser, tTable )
+	return tTable[sUser]
 end
 
-function isthere_key(key, tabl)
-	local q = string.lower(key)
-	for k, v in ipairs(tabl) do
-	w = string.lower(v)
+function isthere_key( key, tTable )
+	local q = key:lower()
+	for k, v in ipairs( tabl ) do
+	w = v:lower()
```

Comment:
The variable `w` is unnecessary here. Just check `q == v:lower()` in the next statement; and that's it.